soccer_ipm/soccer_ipm/msgs/ball.py

```diff
@@ -0,0 +1,53 @@
+from ipm_interfaces.msg import Point2DStamped
```

Comment:
Would need to add Copyright statements to all files for CI to pass since copyright checks aren't skipped anymore.